v1/lib/core/DocsLayout.js

```diff
@@ -76,6 +76,34 @@ class DocsLayout extends React.Component {
         language={metadata.language}
         version={metadata.version}
         metadata={metadata}>
+        {
+          <script
+            dangerouslySetInnerHTML={{
+              __html: `
+                function openConfig(num) {
+                  var i, tabContent, tabLinks;
+                  tabContent = document.getElementsByClassName("tabcontent");
+                  for (i = 0; i < tabContent.length; i++) {
+                      tabContent[i].style.display = "none";
+                  }
+                  tabLinks = document.getElementsByClassName("tablinks");
```

Comment:
Let's specificly check if tabLinks exist. There is a chance that `tabLinks[num]` is undefined and `tabLinks[num].className` is accessing property of undefined which yields an error. Similar to any below codes